test/src/main/java/org/apache/accumulo/test/metrics/MetricsIT.java

```diff
@@ -114,7 +115,7 @@ public void confirmMetricsPublished() throws Exception {
             METRICS_SCAN_RESERVATION_TOTAL_TIMER,
             METRICS_SCAN_RESERVATION_WRITEOUT_TIMER,
             METRICS_SCAN_TABLET_METADATA_CACHE);
-    // @formatter:on
+    // formatter:on
```

Comment:
I'm not sure why this changed, but it has to have the starting character for the formatter to respect it.

```suggestion
    // @formatter:on
```